easybuild/easyconfigs/l/libunwind/libunwind-1.1-foss-2016a.eb

```diff
@@ -19,6 +19,12 @@ checksums = [
     'fb4ea2f6fbbe45bf032cd36e586883ce',     # libunwind-1.1.tar.gz
 ]
 
+dependencies = [
+    ('XZ', '5.2.2'),
+]
+
+preconfigopts = 'export LIBS="$LIBS -llzma" && '
```

Comment:
worth mentioning: easier workaround than patching the broken Makefile, cfr. https://bugs.gentoo.org/show_bug.cgi?id=444050